pkg/netpol/eval/exposure.go

```diff
@@ -26,13 +26,19 @@ import (
 // - generated representative peers are unique; i.e. if different rules (e.g in different policies or different directions)
 // have same selectors, one representative peer is generated to represent both.
 // - note that :
-// - if the rule's namespaceSelector is nil, then the representative pod is created in the policy's Namespace (as it is a real namespace)
+// - this func generates only representative pods (does not generate representative namespaces);
+// each representative pod stores in its data its matching namespaceSelector and podSelector
+// - if the rule's namespaceSelector is nil and the policy is a namespace scoped (NetworkPolicy), then the representative pod is created in
+// the policy's Namespace (as it is a real namespace)
+// - if the rule's namespaceSelector is nil and the policy is cluster-scoped (ANP or BANP), then the representative pod will
+// store an empty namespaceSelector (matches all namespaces in the cluster) in its data
```

Comment:
is this reflected in the output as "all pods with labels... in all namespaces? "

Reply:
yes (as previously)